src/compiler/crystal/tools/doc/html/sitemap.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <% types.each do |type| %><url>
+    <loc><%= ::Path.posix(base_url, type.path) %></loc>
```

Comment:
Is there a reason to use `Path` used instead of `URI`?

Reply:
`Path#join` is more specific to this use case. `URI#resolve` would work as well but it's not necessary to parse and resolve these URIs because we know that `type.path` is relative and only needs to be appended to `base_url`. Thus it's also a bit more efficient.

Reply:
> It not a valid path value.

Of course, it is a perfectly valid posix path (although not normalized). That's why this works.

But sure, I can switch to `URI#resolve`.

Reply:
My bad. I just would never go into Path for a URL. It's fine to leave it as-is. As you prefer.